packages/shared/lib/services/sync/job.service.ts

```diff
@@ -100,7 +100,7 @@ export const updateLatestJobSyncStatus = async (sync_id: string, status: SyncSta
  * @desc grab any existing results and add them to the current
  */
 export const updateSyncJobResult = async (id: number, result: SyncResultByModel, model: string): Promise<SyncJob> => {
-    return db.knex.transaction(async (trx) => {
+    return await db.knex.transaction(async (trx) => {
```

Comment:
why is it necessary? the calls to `updateSyncJobResult` are themselves awaited

Reply:
I don't think there is an issue here, I just wanted to be sure it's awaited somewhere. If on the other end someone is doing `void updateSyncJobResult()` then nobody is awaiting which can be interpreted differently by knex 
https://knexjs.org/guide/transactions.html